cmd/install/addon/cmd.go

```diff
@@ -197,61 +195,36 @@ func run(cmd *cobra.Command, argv []string) {
 		parameters.Each(func(param *cmv1.AddOnParameter) bool {
 			var val string
 			var hasVal bool
+			input := &interactive.Input{
+				Question: param.Name(),
+				Help:     fmt.Sprintf("%s: %s", param.ID(), param.Description()),
+				Required: param.Required(),
+			}
+			options, ok := param.GetOptions()
+			if ok {
+				input.Options = []string{}
+				for _, opt := range options {
```

Comment:
NP: if there are no options, `param.Options` will return a nil array, so you could use 
```suggestion
				for _, opt := range param.Options {
```
as a nil array will be a nop for range, and there is no need to initialize `input.Options` as it already has the zero value when the input is created.